Cannot update chromedriver: Cannot read properties of null (reading 'toString')

I'm on chromedriver `97.0.0`, but running `npm update chromedriver` fails with the following:

```
npm ERR! TypeError: Cannot read properties of null (reading 'toString')
npm ERR!     at isEmulatedRosettaEnvironment (/Users/nicolas/proj/node_modules/chromedriver/install.js:359:14)
npm ERR!     at getMacOsRealArch (/Users/nicolas/proj/node_modules/chromedriver/install.js:343:35)
npm ERR!     at validatePlatform (/Users/nicolas/proj/node_modules/chromedriver/install.js:89:25)
npm ERR!     at Object.<anonymous> (/Users/nicolas/proj/node_modules/chromedriver/install.js:34:18)
npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1103:14)
npm ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:981:32)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:822:12)
npm ERR!     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
npm ERR!     at node:internal/main/run_main_module:17:47
```

I'm on macOS Monterey 12.3 (Intel), node `v16.14.0`, npm `8.3.1`

--- a/install.js
+++ b/install.js
@@ -355,9 +355,20 @@ function isEmulatedRosettaEnvironment() {
   const archName = child_process.spawnSync('uname', ['-m']).stdout.toString().trim();
 
   if (archName === 'x86_64') {
-    const processTranslated = child_process.spawnSync('sysctl', ['-in', 'sysctl.proc_translated'])
-      .stdout.toString()
-      .trim();
+    const proc = child_process.spawnSync('sysctl', ['-in', 'sysctl.proc_translated']);
+
+    // When run with `-in`, the return code is 0 even if there is no `sysctl.proc_translated`
+    if(proc.status) {
+        throw new Error('Unexpected return code from sysctl: ' + proc.status);
+    }
+
+    // If there is no `sysctl.proc_translated` (i.e. not rosetta) then nothing is printed to
+    // stdout
+    if(!proc.stdout) {
+        return false
+    }
+
+    const processTranslated = proc.stdout.toString().trim();
 
     return processTranslated === '1';
   }

